client/components/app.jsx

```diff
@@ -1,15 +1,40 @@
 import React from 'react';
 import Header from './header';
 import ProductList from './product-list';
+import ProductDetails from './product-details';
 
 export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      view: {
+        name: 'catalog',
+        params: {}
+      }
+    };
+  }
+
+  setView = (name, params) => {
+    this.setState({
+      view: {
+        name,
+        params
+      }
+    });
+  }
 
   render = () => {
     return (
-      <>
+      <div className="bg-light">
         <Header />
-        <ProductList />
-      </>
+        {
+          this.state.view.name === 'catalog'
+            ? <ProductList setView={this.setView} />
+            : <ProductDetails setView={this.setView} viewParams={this.state.view.params} />
+        }
+      </div>
     );
+
+    // return <ProductDetails productId={6} />;
```

Comment:
Please do not turn in commented out code.

Reply:
Resolved in 9e60879